catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -518,6 +523,17 @@ define([
                 filters.push(typeFilter)
             }
 
+            var text = this.basicText.currentView.model.getValue()[0];
+
+            if (filters.length == 0) {
+                text = '*'
```

Comment:
If there are no filters yet but someone had typed something into the search box (eg, `foo`) wouldn't this logic see that there were no filters yet and then overwrite the actual search term with '*'?